modules/library/metadata/src/main/java/org/geotools/measure/BaseUnitFormatter.java

```diff
@@ -16,12 +16,13 @@
  */
 package org.geotools.measure;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.io.IOException;
+import java.text.ParsePosition;
+import java.util.*;
```

Comment:
This will (should) fail the PMC checks as we don't allow wildcard imports.

Reply:
I ran mvn spotless:apply. Does it not change that ?

Reply:
Change my IDE setting